### PR TITLE
feat(headerbar)

### PR DESCRIPTION
**Pull request by adeelshahid**
**Progress**

- [x] integrate at code level
- [x] styles missing from default imports
- [x] verify / test HeaderBar API integration. [PR #109](https://github.com/dhis2/ui/pull/109) should fix it for dev. environments.
- [x] translate HeaderBar app title

### CODE REVIEW

File: packages/app/src/components/App.js [hunk 1]
```diff
@@ -85,32 +87,34 @@ export class App extends Component {
 
         return (
             <Fragment>
-                <div className="app">
-                    <div className="item1 headerbar">Headerbar</div>
-                    <div className="item2 visualization-type-selector">
-                        <VisualizationTypeSelector />
+                <UI>
+                    <HeaderBar appName="Data Visualizer" />
```
Comment (varl) [inline packages/app/src/components/App.js]: Very slick.

File: packages/app/public/index.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" dir="ltr">
```
Comment (varl) [inline packages/app/public/index.html]: Should we be able to set this from e.g. `<UI>` or do we have to do this for each app?

File: packages/app/src/components/App.css [hunk 3]
```diff
@@ -33,16 +33,15 @@ body {
...
     grid-gap: 1px;
     grid-template-columns: 250px auto 385px;
-    grid-template-rows: 48px auto auto auto;
     grid-template-areas:
         'headerbar headerbar headerbar'
         'charttype menubar menubar'
         'dimensions chartlayout interpretations'
         'dimensions canvas interpretations';
+    background-color: #aaa;
+    font-family: Roboto, sans-serif;
```
Comment (varl) [inline packages/app/src/components/App.css]: Isn't Roboto as the default font inherited from `<UI>`?

File: packages/app/public/index.html (hunk 2 above)
Comment (adeelshahid) [reply packages/app/public/index.html]: Setting this in `public/index.html` would help us let go of any `React` dependency where we keep a reference to the `<UI /> Component` itself.

It also gives us generic way to update `dir={ltr,rtl}` in all the apps.

Comment (varl) [reply packages/app/public/index.html]: Ok.

File: packages/app/src/components/App.css (hunk 3 above)
Comment (adeelshahid) [reply packages/app/src/components/App.css]: updated, we didn't set any font on `[dir=ltr,rtl] body`. This has been fixed in `d2/ui` hence removing it.

File: package.json [hunk 4]
```diff
@@ -31,5 +31,8 @@
     },
     "workspaces": [
         "packages/*"
-    ]
+    ],
+    "dependencies": {
+        "ui": "github:dhis2/ui-builds"
```
Comment (edoardo) [inline package.json]: Shouldn't this be in the app's package.json?

File: packages/app/src/components/App.js [hunk 5]
```diff
@@ -96,32 +98,34 @@ export class App extends Component {
 
         return (
             <Fragment>
```
Comment (edoardo) [inline packages/app/src/components/App.js]: Do we need `<Fragment>` here now that `<UI>` wraps everything?

File: package.json (hunk 4 above)
Comment (varl) [reply package.json]: Yes.

**Pull request merged**